tests/scripts/install_modules.bat

```diff
@@ -0,0 +1,12 @@
+py -m pip install -U pytest
```

Comment:
This list of install calls can be replaced by `py -m pip install .\tests\requirements\requirements-base.txt` I believe